react-nd/src/components/Topbar/Topbar.module.scss

```diff
@@ -0,0 +1,85 @@
+.header {
+  padding: 2rem 4rem;
+  border-bottom: 2px solid #f1f1f1;
+}
+
+.navWrapper {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+}
+
+.navigation {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1.5rem;
+}
+
+.links {
+  display: flex;
+  gap: 2rem;
+  color: #323232;
+  font-size: 1.2rem;
+  font-weight: 500;
+  text-decoration: none;
+}
+
+.burgerButton {
+  background-color: inherit;
+  color: #8056eb;
+  display: none;
+  font-size: 2rem;
+
+  &:active {
+    color: #323232;
+  }
+}
+
+@media (max-width: 900px) {
+
+  .header {
+    padding: 2rem;
+  }
+
+  .navWrapper {
+    display: block;
+  }
+
+  .burgerButton {
+    display: flex;
+    position: absolute;
+    right: 1rem;
+    top: 2rem;
+  }
+
+  .navigation {
+    padding-top: 1rem;
+    flex-direction: column;
+    height: 0;
+    opacity: 0;
+    overflow: hidden;
+  }
+
+  .burgerOpen {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    opacity: 1;
+    transition: 0.5s ease-in-out;
+  }
+
+  .links {
+    flex-direction: column;
+    text-align: center;
+  }
+}
+
+@media (max-width: 576px) {
+
+  .navbar__burger-btn {
```

Comment:
nėra tikslo naudoti bem standarto, nes visi  klasių pavadinimai šifruojami - sutapimų nebus, dėl to ir BEM klasių vardinimas nebeturi prasmės